File: src/test/java/test/ie/just/another/java/repo/ReverseStringTest.java
package test.ie.just.another.java.repo;

import ie.just.another.java.repo.ReverseString;
import static org.junit.Assert.*;
import org.junit.Test;

public class ReverseStringTest {

    @Test
    public void reverseStringTest() {
        String result = ReverseString.reverseString("abcdefg-123456789");
        assertEquals(result, "987654321-gfedcba");
    }

}

File: src/test/scala/test/ie/just/another/scala/repo/ReverseStringTest.scala
package test.ie.just.another.scala.repo

import ie.just.another.scala.repo.ReverseString
import org.scalatest.{FlatSpec, MustMatchers}

class ReverseStringTest extends FlatSpec with MustMatchers {

  "ReverseString" should "reverse a string" in {
    ReverseString.reverseString("abcdefg-123456789") mustBe "987654321-gfedcba"
  }

}
